src/components/chat/comment-card.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from "@testing-library/react";
+import { Provider } from "mobx-react";
+import React from "react";
+import { UserModelType } from "../../models/stores/user";
+import { ENavTab } from "../../models/view/nav-tabs";
+import { CommentCard } from "./comment-card";
+
+describe("CommentCard", () => {
+
+  const stores = { ui: { activeNavTab: ENavTab.kMyWork } };
+  const activeNavTab = stores.ui.activeNavTab;
+  const testUser  = {  id: "0",
+                      name: "Test Teacher",
+                    } as UserModelType;
+
+  it("should render successfully", () => {
+    render((
+      <Provider stores={stores}>
+        <CommentCard activeNavTab={activeNavTab} />
+      </Provider>
+    ));
+    expect(screen.getByTestId("comment-card")).toBeInTheDocument();
+    expect(screen.getByTestId("comment-card")).toHaveClass(activeNavTab);
+    expect(screen.getByTestId("comment-card-header")).toBeInTheDocument();
+  });
+  it("should show the correct header icon when there are no comments", () => {
+    const postedComments:any = [];
```

Comment:
If you had the `ICommentData` interface suggested above you could use it here instead of `any`.